server/aws-lsp-codewhisperer/src/shared/telemetryUtils.ts

```diff
@@ -47,8 +47,9 @@ export const getUserAgent = (initializeParams: InitializeParams, serverInfo?: Se
 }
 
 const IDE_CATEGORY_MAP: { [key: string]: IdeCategory } = {
+    // TODO: VSCode key needs to change for getting the correct coefficient value for inline
     'AmazonQ-For-VSCode': 'VSCODE',
-    'Amazon-Q-For-JetBrains': 'JETBRAINS',
+    'Amazon Q For JetBrains': 'JETBRAINS',
```

Comment:
What is this inconsistency caused by? Cannot we pass extension name in `clientInfo` following the same format as for other IDEs?

Reply:
The extension name for JetBrains is "Amazon Q For JetBrains", with spaces, not sure why the version with hyphens are added there, this is tested with Jb talking to flare, 
For Eclipse and VS, they should be already talking to flare so assuming the names are correct, for VSC I'm not sure, it will be determined when we decide to move VSC inline to flare.